pkg/etcdenvvar/etcd_env.go

```diff
@@ -273,3 +278,27 @@ func getUnsupportedArch(envVarContext envVarContext) (map[string]string, error)
 		"ETCD_UNSUPPORTED_ARCH": arch,
 	}, nil
 }
+
+func getCipherSuites(envVarContext envVarContext) (map[string]string, error) {
+	var observedConfig map[string]interface{}
+	if err := yaml.Unmarshal(envVarContext.spec.ObservedConfig.Raw, &observedConfig); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal the observedConfig: %w", err)
+	}
+	observedCipherSuites, _, err := unstructured.NestedStringSlice(observedConfig, "servingInfo", "cipherSuites")
+	if err != nil {
+		return nil, fmt.Errorf("couldn't get cipherSuites from observedConfig: %w", err)
+	}
+	observedCiphersWhitelist := tlshelpers.WhitelistEtcdCipherSuites(observedCipherSuites)
+
+	var validCipherSuites []string
+	switch {
+	case len(observedCiphersWhitelist) > 0:
+		validCipherSuites = observedCiphersWhitelist
+	default:
```

Comment:
Rather than have a default, you could require a config be set.  See https://github.com/openshift/cluster-kube-apiserver-operator/blob/eafc4009be6dfac5d46a5ee5b203142bfb8a73ae/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L112 for inspiration.  That keeps all the logic for "what ciphers should I use" in one spot instead of two.